packages/seacas/applications/ejoin/EJ_Version.h

```diff
@@ -34,8 +34,8 @@
 
 static char const *qainfo[] = {
     "ejoin",
-    "2018/07/26",
-    "1.5.2",
+    "2019/05/06",
+    "1.5.3",
```

Comment:
How can the date be newer but the version be older?

Reply:
My math shows both are newer...

Reply:
Must have been an optical illusion. I could have sworn that it was '1.5.3' on the left and '1.5.2' on the right.  Sorry about that.